keps/prod-readiness/sig-node/1769.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 1769
+alpha:
+  - "@deads2k"
```

Comment:
the syntax is wrong here and is failing the bot.

should be the following:

```
kep-number: 1769
alpha:
  approver: "@deads2k"
```

Reply:
@derekwaynecarr It was corrected. Many thanks.